R/pkg_ref.R

```diff
@@ -24,81 +60,219 @@ as.pkg_ref.default <- function(x, ...) {
 
 
 #' @export
-as.pkg_ref.pkg_ref <- function(x, ...) {
+as_pkg_ref.pkg_ref <- function(x, ...) {
   x
 }
 
 
 
-#' @importFrom utils installed.packages available.packages
+#' @importFrom utils installed.packages available.packages packageVersion
 #' @importFrom xml2 read_html
 #' @export
-as.pkg_ref.character <- function(x, repos = getOption("repos"), ...) {
-  ip <- utils::installed.packages()
-  ap <- utils::available.packages(repos = repos)
+as_pkg_ref.character <- function(x, repos = getOption("repos"), ...) {
```

Comment:
The function assumes 

"xxx" : an installed or remote R package
"path/xxx": an R package source

Could we clarify this and provide some examples in the documentation?

Reply:
This is documented in `as_pkg_ref` (which then dispatches to `as_pkg_ref.character`)

https://github.com/pharmaR/riskmetric/blob/03ebf2bbaf5d8651ad034f41a7e72abac5f9fc89/R/pkg_ref.R#L33-L49

Reply:
Yes, I hope we could make the documentation more clear which pattern of character is corresponding to which type of source.